### PR TITLE
Improve styling for HTML reports

### PR DESCRIPTION
**Pull request by pastelsky**
1. Enable text wrapping and use more subtle colors for ANSI text in errors view —
Screenshot 2022-11-30 at 12 46 50 AM

2. Use subtle shadows on image and video elements, more grabbable handle —
Screenshot 2022-11-30 at 12 45 50 AM

3. The Image Mismatch section is now displayed first (just after errors) as its more often than not what you want to review first if present.

### CODE REVIEW

File: packages/html-reporter/src/testResultView.tsx [hunk 1]
```diff
@@ -106,6 +106,10 @@ export const TestResultView: React.FC<{
       </AutoChip>
     )}
 
+    {!!result.steps.length && <AutoChip header='Test Steps'>
```
Comment (pavelfeldman) [inline packages/html-reporter/src/testResultView.tsx:109]: Why do we show this twice now?

File: packages/html-reporter/src/testResultView.css [hunk 2]
```diff
@@ -40,7 +41,7 @@
 }
 
 .test-result-error-message {
-  white-space: pre;
+  white-space: pre-wrap;
```
Comment (pavelfeldman) [inline packages/html-reporter/src/testResultView.css:44]: I don't think you can do this due to error pointer:
wrap

File: packages/html-reporter/src/common.css [hunk 3]
```diff
@@ -16,11 +16,9 @@
 
 :root {
   --box-shadow: rgba(0, 0, 0, 0.133) 0px 1.6px 3.6px 0px, rgba(0, 0, 0, 0.11) 0px 0.3px 0.9px 0px;
-  --box-shadow-thick: rgb(0 0 0 / 10%) 0px 1.8px 1.9px,
-    rgb(0 0 0 / 15%) 0px 6.1px 6.3px,
-    rgb(0 0 0 / 10%) 0px -2px 4px,
-    rgb(0 0 0 / 15%) 0px -6.1px 12px,
-    rgb(0 0 0 / 25%) 0px 6px 12px;
+  --box-shadow-thick: 0.4px 0.4px 1.4px rgba(0, 0, 0, 0.07),
```
Comment (pavelfeldman) [inline packages/html-reporter/src/common.css:19]: lgtm

File: packages/html-reporter/src/testResultView.tsx [hunk 4]
```diff
@@ -173,20 +177,20 @@ const ErrorMessage: React.FC<{
 
 const ansiColors = {
   0: '#000',
-  1: '#C00',
-  2: '#0C0',
-  3: '#C50',
-  4: '#00C',
-  5: '#C0C',
-  6: '#0CC',
+  1: '#d53627',
```
Comment (pavelfeldman) [inline packages/html-reporter/src/testResultView.tsx:180]: lgtm

File: packages/html-reporter/src/testResultView.css (hunk 2 above)
Comment (pastelsky) [reply packages/html-reporter/src/testResultView.css:44]: Ah, right. Bummer!
It'd be great if playwright had proper typed diagnostics for these errors. Currently, they're just strings, so we have to unfortunately dump them as code in HTML reports.

File: packages/html-reporter/src/testResultView.tsx (hunk 1 above)
Comment (pastelsky) [reply packages/html-reporter/src/testResultView.tsx:109]: Looks like a problem with my git patch. Removed.

**Comment by pastelsky** [comment]
Thanks for reviewing @pavelfeldman . Is there a better develop to iterate on the HTML reports react page? Currently I need to manually run `yarn build` in the package, and start an integration test to preview my changes.

**Comment by pastelsky** [comment]
I wonder why the CI checks are stuck?

**Comment by mxschmitt** [comment]
> I wonder why the CI checks are stuck?

We have tests for the html report which are failing: https://github.com/microsoft/playwright/actions/runs/3592702190/jobs/6048732780#step:9:167 after your changes.

**Comment by pavelfeldman** [comment]
Looks like the tests failed. Do you want to look at the failures?

**Comment by pavelfeldman** [comment]
Closing as stake, but please feel free to re-post of reopen if you get back to it!

**Pull request closed without merging**